composer.json

```diff
@@ -18,14 +18,19 @@
             "name": "Stefan Kleff",
             "email": "s.kleff@goalio.de",
             "homepage": "http://www.goalio.de"
+        },
+        {
+            "name": "Bas Kamer",
+            "email": "baskamer@gmail.com",
+            "homepage": "https://bushbaby.nl"
         }
     ],
     "require": {
-        "php": "^5.5 || ^7.0",
+        "php": "^7.2",
         "laminas/laminas-eventmanager": "^2.6 || ^3.0",
```

Comment:
Have a look at SlmQueue. I raised all depencies to the minimum version I could find that actually support PHP 7.2. I did that also for the dev-dependencies.

Little trick I used to do this fast is to look at the HTML, and search for `PHP 7.`. For example in view-source:https://github.com/laminas/laminas-eventmanager/releases.